### DISCUSSION TITLE
代码报错、get之后有一部分代码报错

### DISCUSSION THREAD

**Discussion by afskalsfk** [TelsaCoin/TelsaVideo#46] [Q&A]
某些方法和基础组件查不到  如图  请问这个怎么解决
<img width="955" alt="截屏2022-12-13 10 21 13" src="https://user-images.githubusercontent.com/26595286/207210559-544e4217-1881-45d5-b916-a001734e42a0.png">

**Comment by charliewwdev** [comment]
版本问题
